apps/core/models/core.py

```diff
@@ -1,7 +1,17 @@
+import math
+
+from django.core.validators import ValidationError
 from django.db import models
 
 
+def validate_aspect_ratio(image):
+    ratio = 1
+    if not math.isclose(image.width / image.height, ratio, rel_tol=1e-6):
+        raise ValidationError("The aspect ratio is not correct. The aspect ratio should be: " + str(ratio))
```

Comment:
Might be best to move this up to a utils file or sth, as this is now used and defined in multiple locations.